### PR TITLE
Removes low memory check when deciding if to clean disk

### PR DESCRIPTION
**Pull request by OferMania**
- Also command-line parametrizes other variables used by LogCleaner
- We introduce ratio to the memory check

### CODE REVIEW

File: src/LogCleaner.cc [hunk 1]
```diff
@@ -759,19 +768,19 @@ LogCleaner::Balancer::isMemoryLow(CleanerThreadState* thread)
     const int T = cleaner->segmentManager.getMemoryUtilization();
     const int L = cleaner->cleanableSegments.getLiveObjectUtilization();
 
+    // TODO: When we exceed dangerThreshold, we should probably
+    // do something, but for now, we warn the user
+    if (T >= cleaner->dangerThreshold) {
+        LOG(WARNING, "Memory Utilization (%d) exceeds Danger Threshold", T);
```
Comment (james-munson) [inline src/LogCleaner.cc:774]: How often is this checked, and potentially logging?  Is it another log flooder?

File: src/ServerMain.cc [hunk 2]
```diff
@@ -196,6 +206,21 @@ main(int argc, char *argv[])
                &config.backup.maxRecoveryReplicas)->default_value(20),
              "Maximum number of replicas any given master recovery will buffer "
              "in memory.")
+            ("minDiskUtilization",
+             ProgramOptions::value<uint32_t>(
+               &config.backup.minDiskUtilization)->default_value(95),
```
Comment (james-munson) [inline src/ServerMain.cc:211]: I'm a little surprised these parameters kept the old high default values.  I though part of the issue was that it didn't start cleaning until rather late in the game.

File: src/LogCleaner.cc (hunk 1 above)
Comment (OferMania) [reply src/LogCleaner.cc:774]: It isn't, at least not so far, based on the runs Dan & I are doing.

File: src/ServerMain.cc (hunk 2 above)
Comment (OferMania) [reply src/ServerMain.cc:211]: I'm only keeping the older defaults until we figure out what the "new defaults" should be. Then I can change this PR to reflect that.

**Pull request closed without merging**